src/components/charts/TradesLog.vue

```diff
@@ -62,10 +62,18 @@ export default class TradesLogChart extends Vue {
       .sort((a, b) => (a.close_timestamp > b.close_timestamp ? 1 : -1));
     for (let i = 0, len = sortedTrades.length; i < len; i += 1) {
       const trade = sortedTrades[i];
-      const entry = [i, (trade.profit_ratio * 100).toFixed(2), trade.pair, trade.botName];
+      const closeDate = trade.close_date
+        .substring(0, trade.close_date.length - 6)
```

Comment:
Rather than doing "odd" math here - i'd suggest using `trade.close_timestamp` - and using one of the formatters in `formatters.ts` - either `timestampToDateString` (to get the date) - or `timestampmsWithTimezone` (to get the full timestamp, not just the date).